scripts/bintray/deploy_rpm.bash

```diff
@@ -1,37 +1,80 @@
 #!/bin/bash -x
 
 set -eEuo pipefail
-set -x
 
-# TODO:
-# This is a hacky, hacky script to keep going
-# 
+function show_help {
+    cat - <<HEREDOC
+Usage:
 
-function usage {
-    echo "Usage: ${0##*/} <USER> <PASSWORD> <VERSION> [...other ignored]"
+    ${0##*/} -u USERNAME -p PASSWORD -v VERSION -r RPM_LIFECYCLE -c DEB_COMPONENT [-d]
+
+    where:
+        -d            - debug mode
+        RPM_LIFECYCLE - the label identyfuing repository
+                        (e.g. stable, testing, unstable). Defaults to 'dev'.
+
+HEREDOC
+}
+
+function assert_vars_set {
+    for n in "$@"; do
+        if [[ -n "$n" ]] && [[ -z "${!n}" ]]; then
+            >&2 show_help
+            >&2 echo "ERROR: value for ${n} was not provided"
+            exit 1
+        fi
```

Comment:
Purely an observation, but this version of assert_vars_set doesn't log out the value in debug mode.